src/download.ts

```diff
@@ -93,8 +75,6 @@ function getDetachedSubmodules(submodules: string[]): string[] {
 }
 
 /**
- * Mappes file ids to their export paths.
- *
  * @returns File ids mapped to their export path.
```

Comment:
Changes here are inconsistent throughout the PR - the description is removed, but the return description is kept. Please keep both.